workers/unity/Assets/Playground/Scripts/Player/ProcessScoresSystem.cs

```diff
@@ -0,0 +1,36 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Collections;
+using Unity.Entities;
+
+namespace Playground
+{
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    class ProcessScoresSystem : ComponentSystem
+    {
+        private struct ScoringData
+        {
+            public readonly int Length;
+            public ComponentDataArray<SpatialOSLauncher> Launcher;
```

Comment:
Might want to add that you need authority too. There is a race on receiving a command request and then losing authority. Although really we should handle that in core if we don't, as that can only troll users.

Reply:
However there is still an argument for adding it to specify intent.